patches/api/0464-Add-paper-version-util-class.patch

```diff
@@ -0,0 +1,188 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: masmc05 <masmc05@gmail.com>
+Date: Fri, 16 Feb 2024 14:13:30 +0200
+Subject: [PATCH] Add paper version util class
+
+
+diff --git a/src/main/java/io/papermc/paper/util/ServerInfo.java b/src/main/java/io/papermc/paper/util/ServerInfo.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..cf39052c7d9f152313762e928fb49c5e3cd22dbc
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/util/ServerInfo.java
+@@ -0,0 +1,98 @@
++package io.papermc.paper.util;
++
++import net.kyori.adventure.key.Key;
++import net.kyori.adventure.util.Services;
++import org.bukkit.UnsafeValues;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * A utility class to get information about the server
++ * Works even before Bukkit is initialized (e.g. on bootstrap)
++ */
++public class ServerInfo {
```

Comment:
Is there a reason why you don't just make the class abstract?

Reply:
Why would I?

Reply:
Yeah, seems like you misunderstood something